rs/rosetta-api/icrc1/tests/upgrade_downgrade.rs

```diff
@@ -145,25 +148,6 @@ fn install_index_ng(env: &StateMachine, init_arg: IndexInitArg) -> CanisterId {
     .unwrap()
 }
 
-fn ledger_mainnet_wasm() -> Vec<u8> {
```

Comment:
Can we remove this method from tests.rs and use the method from common there as well?

Reply:
There's no `ledger_mainnet_wasm` method in [`rs/rosetta-api/icrc1/tests/tests.rs`](https://sourcegraph.com/github.com/dfinity/ic@b92f832856043a52023f05d5701e84f2511f0bdc/-/blob/rs/rosetta-api/icrc1/tests/tests.rs), did you mean some other `tests.rs`? Similar methods do exist in other places under `rs/rosetta-api`, and it could make sense to consolidate them, but I would do that in a separate ticket.

Reply:
Right, I was looking at ledger/tests/tests.rs... Sorry, nevermind:)